torch-sys/libtch/torch_api.cpp

```diff
@@ -1408,11 +1408,18 @@ ivalue ati_object_method_(ivalue i, char *method_name, ivalue *ivalues, int niva
   return nullptr;
 }
 
+ivalue ati_object_getattr_(ivalue i, char *attr_name) {
+  PROTECT(
+    torch::jit::IValue output  = i->toObjectRef().getAttr(attr_name);
+    return new torch::jit::IValue(output);
+  )
+  return nullptr;
+}
+
 ivalue ati_clone(ivalue i) {
   PROTECT(
     return new torch::jit::IValue(*i);
   )
-  return nullptr;
```

Comment:
Why was this removed? Could you put it back as it's useful to avoid warnings for some compilers?